src/common/utils/utils.ts

```diff
@@ -64,7 +64,7 @@ export function getEstimatedFee(): Promise<SatoshiBig> {
       bridgeService.getQueuedPegoutsCount(),
     ])
       .then(([nextPegoutCost, pegoutQueueCount]) => {
-        const estimatedFee = nextPegoutCost / (pegoutQueueCount + 1);
+        const estimatedFee = nextPegoutCost / (pegoutQueueCount + 1n);
```

Comment:
``
```suggestion
        const estimatedFee = (nextPegoutCost ? 1 : nextPegoutCost) / ((pegoutQueueCount ? 0 : pegoutQueueCount) + 1n);
```

Reply:
I get your point but `nextPegoutCost` should never be 0 according to [RSKIP385](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP385.md). If we want to be extra cautious it is not as simple as using a default value, we must do the calc.